sql/src/test/java/io/crate/executor/transport/TransportExecutorTest.java

```diff
@@ -195,27 +193,45 @@ public void testESMultiGet() throws Exception {
     }
 
     @Test
-    public void testESSearchTask() throws Exception {
+    public void testQTFTask() throws Exception {
+        // select id, name from characters;
         setup.setUpCharacters();
-
         DocTableInfo characters = docSchemaInfo.getTableInfo("characters");
+        ReferenceInfo docIdRefInfo = characters.getReferenceInfo(new ColumnIdent(DocSysColumns.DOCID.name()));
+        List<Symbol> collectSymbols = Lists.<Symbol>newArrayList(new Reference(docIdRefInfo));
+        List<Symbol> outputSymbols = Lists.<Symbol>newArrayList(idRef, nameRef);
 
-        ESQueryThenFetchNode node = new ESQueryThenFetchNode(
-                characters.getRouting(WhereClause.MATCH_ALL, null),
-                Arrays.<Symbol>asList(idRef, nameRef),
-                Arrays.<Symbol>asList(nameRef, idRef),
-                new boolean[]{false, false},
-                new Boolean[]{null, null},
-                null, null, WhereClause.MATCH_ALL,
-                null
+        Planner.Context ctx = new Planner.Context();
+
+        CollectNode collectNode = PlanNodeBuilder.collect(
+                characters,
+                ctx,
+                WhereClause.MATCH_ALL,
+                collectSymbols,
+                ImmutableList.<Projection>of(),
+                null,
```

Comment:
ok I'v just saw your new ordered test, sry. so just validate that the test is deterministic ;)